### ISSUE TITLE
Installation issue

### ISSUE THREAD

**Issue by ramsinghdayal** [joferkington/python-geoprobe#2]
Is python-geoprobe is available for cp36/win32 .whl file?

**Comment by joferkington** [comment]
No, but the module is platform independent.  The source distribution should work fine. Let me know if it's doesn't for some reason.
